sklearn/linear_model/randomized_l1.py

```diff
@@ -386,7 +386,7 @@ class RandomizedLogisticRegression(BaseRandomizedLinearModel):
 
     Parameters
     ----------
-    C : float, optional, default=1
+    C : float or array, optional, default=1
```

Comment:
You should indicate the behavior when C is an array: what are the different values of the array used for.